src/modules/pages/components/IncorporationPage/IncorporationPage.tsx

```diff
@@ -55,7 +51,31 @@ const IncorporationPage = () => {
   }, []);
 
   const handlePay = useCallback(() => {
-    setActiveStageId(StagesEnum.Processing);
+    setMotions((prevState) => [
```

Comment:
Please refactor this without the timeout, it is really only useful for reviewing and won't be needed for the interface.